test/libsolidity/semanticTests/strings/return_string.sol

```diff
@@ -0,0 +1,18 @@
+contract Main {
+    string public s;
+    function set(string calldata _s) external {
+        s = _s;
+    }
+    function get1() public returns (string memory r) {
+        return s;
+    }
+    function get2() public returns (string memory r) {
+        r = s;
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// get1() -> 0x20, 0
```

Comment:
This is missing `set(string): 0x20, 5, "Julia"`

Reply:
Update : Added `set(string)` function to the .sol file and pushed changes